sections/03_categorize.md

```diff
@@ -278,77 +276,115 @@ Methods to accomplish more with little high-quality labeled data are also being
 applied in other domains and may also be adapted to this challenge, e.g. data
 programming [@arxiv:1605.07723]. In data programming, noisy automated labeling
 functions are integrated. Numerous writers have described data as the new oil
-[@url:http://ana.blogs.com/maestros/2006/11/data_is_the_new.html, @url:https://medium.com/twenty-one-hundred/data-is-the-new-oil-a-ludicrous-proposition-1d91bba4f294].
+[@url:http://ana.blogs.com/maestros/2006/11/data_is_the_new.html,
+@url:https://medium.com/twenty-one-hundred/data-is-the-new-oil-a-ludicrous-proposition-1d91bba4f294].
 The idea behind this metaphor is that data are available in large quantities,
 valuable once refined, and the underlying resource that will enable a
 data-driven revolution in how work is done. Contrasting with this perspective,
 Ratner, Bach, and Ré described labeled training data as "The _New_ New Oil"
 [@url:http://hazyresearch.github.io/snorkel/blog/weak_supervision.html]. In this
-framing, data are abundant and not a scarce resource. Instead, new approaches
-to solve problems arise when labeled training data become sufficient to enable
+framing, data are abundant and not a scarce resource. Instead, new approaches to
+solve problems arise when labeled training data become sufficient to enable
 them. Based on our review of research on deep learning methods to categorize
 disease, the latter framing rings true.
 
-We expect improved methods for domains with limited data to play an important
-role if deep learning is going to transform how we categorize states of human
-health. We don't expect that deep learning methods will replace expert review.
-We expect them to complement expert review by allowing more efficient use of the
-costly practice of manual annotation.
+In addition to methodological improvements, a robust culture of data sharing -
+and in particular the sharing of high-quality labeled datasets - would do much
+to speed advances in this domain. The cultural barriers are perhaps best
+captured by the implications of using the term "research parasite" to describe
+scientists who use data from other researchers [@doi:10.1056/NEJMe1516564]. In
+short, a field that honors only discoveries and not the hard work of generating
+useful data will have difficulty encouraging scientists to share their hard-won
+data. Unfortunately, it's precisely those data that would help to power deep
+learning in the domain. Though not a methodological consideration, efforts are
+underway to recognize those who promote an ecosystem of rigorous sharing and
+analysis [@doi:10.1038/ng.3830].
+
+We expect both improved methods and an improved culture of sharing to play an
```

Comment:
Hmm - thought I made a comment on this but don't see it. Can you make this match the text from https://github.com/greenelab/deep-review/blame/master/sections/03_categorize.md#L292